lib/repo/subversion_repository.rb

```diff
@@ -584,25 +579,23 @@ def self.__read_in_authz_file
       return file_content
     end
 
-    # Semi-private class method: Writes out Repository.conf[:REPOSITORY_PERMISSION_FILE]
+    # Semi-private class method
     def self.__write_out_authz_file(authz_file_contents)
       # Check if configuration is in order
-      if Repository.conf[:IS_REPOSITORY_ADMIN].nil?
-        raise ConfigurationError.new("Required config 'IS_REPOSITORY_ADMIN' not set")
+  
+      if !MarkusConfigurator.markus_config_repository_admin?
+        raise NotAuthorityError.new("Unable to write authsz file:  Not in authoritative mode!");
       end
-      if Repository.conf[:REPOSITORY_PERMISSION_FILE].nil?
+
+      if MarkusConfigurator.markus_config_repository_permission_file.nil?
         raise ConfigurationError.new("Required config 'REPOSITORY_PERMISSION_FILE' not set")
       end
-      # If we're not in authoritative mode, bail out
-      if !Repository.conf[:IS_REPOSITORY_ADMIN] # Are we admin?
-        raise NotAuthorityError.new("Unable to write out repo permissions:  Not in authoritative mode!");
-      end
 
-      if !File.exist?(Repository.conf[:REPOSITORY_PERMISSION_FILE])
-        File.open(Repository.conf[:REPOSITORY_PERMISSION_FILE], "w").close() # create file if not existent
+      if !File.exist?(MarkusConfigurator.markus_config_repository_permission_file)
+        File.open(MarkusConfigurator.markus_config_repository_permission_file, "w").close() # create file if not existent
       end
       result = false
-      File.open(Repository.conf[:REPOSITORY_PERMISSION_FILE], "w+") do |auth_file|
+      File.open(MarkusConfigurator.markus_config_repository_permission_file, "w+") do |auth_file|
```

Comment:
Line is too long. [97/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.